mm2src/coins/utxo/utxo_common.rs

```diff
@@ -197,6 +198,48 @@ where
     Ok(result)
 }
 
+pub async fn generate_and_confirm_new_address<Coin, ConfirmAddress>(
+    coin: &Coin,
+    hd_wallet: &Coin::HDWallet,
+    hd_account: &mut Coin::HDAccount,
```

Comment:
why is this not `mut hd_account: Coin::HDAccount` here and everywhere? I didn't see the value being used after being passed

Reply:
We could pass the `hd_account` by value if only the function takes `mut hd_account: HDAccountMut<Coin::HDAccount>`. This has one disadvantage - you can pass the value only once.
In this example, we pass `hd_accounts` twice:
https://github.com/KomodoPlatform/atomicDEX-API/blob/66c43bb15b0c8ed14e4ea2aec34014b284ae3cfa/mm2src/coins/rpc_command/get_new_address.rs#L445-L462
and can modify the function so it can pass the `hd_accounts` as a mutable reference multiple times.

Reply:
alright got it. thanks